src/deploymentModals.jsx

```diff
@@ -73,7 +73,7 @@ export const CleanUpModal = ({ os }) => {
     const actions = [
         <Button key="cleanup-accept"
             variant="primary"
-            isDisabled={buttonLoading || (!deleteTemporaryFiles && !deleteRPMmetadata && !deletePendingDeployments && !deleteRollbackDeployments)}
+            isAriaDisabled={buttonLoading || (!deleteTemporaryFiles && !deleteRPMmetadata && !deletePendingDeployments && !deleteRollbackDeployments)}
```

Comment:
This added line is not executed by any test.

Reply:
This needs some explanation. Why is `isDisabled` now out of fashion? We still use it in cockpit.git. AFAIUI, the aria properties should only be used if the regular properties somehow don't exist -- aria is a hack/workaround, proper semantic HTML is better.

Reply:
@martinpitt it's for accessibility reasons. To quote [Patternfly website](https://www.patternfly.org/components/button/accessibility)

> If a button should be disabled, but remain focusable, it has the `aria-disabled="true"` attribute instead of the standard `disabled` attribute.

Reply:
This feels upside down to me, but I'll bow to the expert :grin:  Thanks!

Reply:
@martinpitt You can try it on their website too so you can see it in action with keyboard navigation

Try to tab to the [disabled buttons here](https://www.patternfly.org/components/button#disabled-buttons) and it won't work, but [aria disabled buttons here](https://www.patternfly.org/components/button#aria-disabled-examples) you can focus

Reply:
Right, but is that desirable? We e.g. don't make the disabled link buttons tab-able on the Overview page either. Maybe that's desired, maybe not. I don't know, so I'll just let you decide. At least I now now what the difference is, thanks!